Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");

```

Comment:
**Suggestion**: The method name and print statement suggest that this method should only print even numbers, but the loop logic does not specifically check if the numbers are even. Ensure the logic matches the method's purpose by checking the number's evenness before printing. <br> **Code Suggestion**: 
 ```
 +        System.out.println("Even numbers:");
 +        for (int i = 1; i < arr.length; i += 2) {
 +            if (arr[i] % 2 == 0) {
 +                System.out.println(arr[i]);
 +            }
 +        }
 ```